frontend/src/lib/v2/DynamicFlow.ts

```diff
@@ -70,12 +216,19 @@ export const TASK_NAME_KEY = 'task_name';
 //    How to handle parallel-for? and list of workers.
 
 export function updateFlowElementsState(
+  layers: string[],
   elems: PipelineFlowElement[],
   executions: Execution[],
   events: Event[],
   artifacts: Artifact[],
-) {
+): PipelineFlowElement[] {
   // IMPORTANT: PipelineFlowElement update is in-place.
```

Comment:
In the current design, the element update is longer in-place. Can you either remove the comment or change the comment? Thank you.

Reply:
Good point! Removed the comment.